.github/workflows/msvc-build.yml

```
@@ -49,12 +50,19 @@
 
     steps:
       - name: Checkout repository
         uses: actions/checkout@v4
       
+      - name: Setup Visual Studio 2017 Build Tools
+        run: |
+          # 下载并安装 Visual Studio 2017 Build Tools
+          Invoke-WebRequest -Uri "https://aka.ms/vs/15/release/vs_buildtools.exe" -OutFile "vs_buildtools.exe"
+          Start-Process -FilePath "vs_buildtools.exe" -ArgumentList "--quiet", "--wait", "--add", "Microsoft.VisualStudio.Workload.VCTools", "--add", "Microsoft.VisualStudio.Component.VC.Tools.x86.x64", "--add", "Microsoft.VisualStudio.Component.Windows10SDK.18362" -NoNewWindow -Wait
+        shell: powershell
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)